app/askforrecommendation/[id]/page.tsx

```diff
@@ -171,55 +173,99 @@ export default function AskForRecommendation() {
-            Copy the message below using the copy icon
-          </li>
+          <li style={{ marginBottom: '8px' }}>Copy the message below using the Button</li>
           <li style={{ marginBottom: '8px' }}>Open your preferred email application</li>
           <li>Paste the message and send it to your desired recommender</li>
         </ol>
-
+        <Box sx={{ display: 'flex', justifyContent: 'center', mb: 3 }}>
+          <Button
+            onClick={copyToClipboard}
+            variant='outlined'
+            color='primary'
+            sx={{
+              borderRadius: '100px',
+              textTransform: 'lowercase',
+              fontFamily: 'Roboto',
+              color: '#FFFFFF',
+              fontSize: '14px',
+              width: 'fit-content',
+              backgroundColor: '#003FE0',
+              '&:hover': {
+                backgroundColor: '#003FE0'
+              }
+            }}
+            startIcon={<ContentCopyIcon />}
+          >
+            Copy the message
+          </Button>
+        </Box>
         <Box
           sx={{
             backgroundColor: 'white',
             p: 3,
             borderRadius: '4px',
             position: 'relative',
             border: '1px solid #e0e0e0',
-            flexWrap: 'wrap'
+            flexWrap: 'wrap',
+            display: 'flex',
+            flexDirection: 'column',
+            width: '100%'
           }}
         >
-          <Typography
+          <Box
             sx={{
-              // whiteSpace: 'pre-wrap',
+              position: 'relative',
+              display: 'flex',
+              alignItems: 'center',
+              justifyContent: 'space-between',
+              wordWrap: 'break-word',
+              overflowWrap: 'break-word',
+              width: '100%',
               fontFamily: 'Lato',
               color: '#333',
               fontSize: '14px',
-              wordWrap: 'break-word'
+              flexGrow: 1
             }}
           >
-            {messageToCopy}
-          </Typography>
+            <Typography
+              sx={{
+                fontFamily: 'Lato',
+                color: '#333',
+                fontSize: '14px',
+                wordWrap: 'break-word',
+                overflowWrap: 'break-word',
+                overflow: 'hidden',
+                textOverflow: 'ellipsis',
+                flexGrow: 1,
+                lineHeight: 1.5
+              }}
+            >
+              {messageToCopy}
+            </Typography>
 
-          <Box
-            onClick={copyToClipboard}
-            sx={{
-              position: 'absolute',
-              right: '12px',
-              top: '12px',
-              cursor: 'pointer',
-              padding: '8px',
-              borderRadius: '4px',
-              '&:hover': {
-                backgroundColor: 'rgba(0, 0, 0, 0.04)'
-              }
-            }}
-          >
-            <ContentCopyIcon sx={{ color: '#666' }} />
+            {/* <Box
```

Comment:
I think it's safe to just delete this, instead of commenting it out.

Reply:
ok consider it deleted